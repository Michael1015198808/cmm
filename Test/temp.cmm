int main() {
    int i = 1;
    a[5 a;
    i = 1;
    iint i = 1;
    int i = 1;
}
