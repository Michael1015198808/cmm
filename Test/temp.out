Error type B at Line 3: Missing "]"
Error type B at Line 5: Missing ";"
Error type B at Line 6: Something wrong with the statement
